src/index.tsx

```diff
@@ -9,6 +9,11 @@ import ReactDOM from "react-dom";
 import App from "./App";
 import "./index.scss";
 import reportWebVitals from "./reportWebVitals";
+import { makeServer } from "./server";
+
+if (process.env.NODE_ENV === "development") {
+  makeServer({ environment: "development" });
+}
```

Comment:
Only run the Mirage server if in the dev environment.

Reply:
Wonder if there's another way to do this that doesn't make the application aware of what env it's running in. But it's just 3 lines.

Reply:
Default inside makeServer();